### PR TITLE
Feature/legionairre nuc 560 transfer icon button component

### PR DESCRIPTION
**Pull request by legionairre**
localhost_8080_components_iconbutton html

### CODE REVIEW

**Comment by legionairre** [comment]
> Can you merge this repo with v0.3.0 branch to get brand new folder structure that includes /core folder

It has been done

File: docs/components/vxpiconbutton.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+}
+```
+
+<IconButtonDoc />
+</DocExampleBox>
+
+### Props
+
+| Name      | Type           | Default | Web | Mobile |
+| --------- | -------------- | ------- | --- | ------ |
```
Comment (zhunor) [inline docs/components/vxpiconbutton.md]: Please add description field here with a little bit explanation to keep consistency with other pages.

File: docs/components/vxpiconbutton.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+| icon      | String         |         | ✔   | ✔      |
+| size      | String, Number | 37      | ✔   | ✔      |
+| isEnabled | Boolean        |         | ✔   | ✔      |
+
+### Events
+
+| Name | Web | Mobile |
```
Comment (zhunor) [inline docs/components/vxpiconbutton.md]: Please add description field here with a little bit explanation to keep consistency with other pages.

File: docs/components/vxpiconbutton.md (hunk 1 above)
Comment (legionairre) [reply docs/components/vxpiconbutton.md]: It has been added.

File: docs/components/vxpiconbutton.md (hunk 2 above)
Comment (legionairre) [reply docs/components/vxpiconbutton.md]: It has been added.

File: docs/components/vxpiconbutton.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+| size      | String, Number | 37      | ✔   | ✔      | Sets Icon size.                           |
+| isEnabled | Boolean        |         | ✔   | ✔      | Sets the VxpIconButton enable or disable. |
+
+### Events
+
+| Name | Web | Mobile | Description                               |
+| ---- | --- | ------ | ----------------------------------------- |
+| tap  | ✔   | ✔      | Emitted when the VxpIconButton is tapped. |
+
+#### Note: The following components to be used from 'nativescript-vue-web'
```
Comment (ahmetceylan) [inline docs/components/vxpiconbutton.md]: please remove nativescript vue web references

File: src/components/VxpIconButton.vue [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+    };
+  },
+  props: {
+    icon: String,
+    size: { type: [Number, String], default: 40 },
+    iconName: String,
+    isEnabled: { type: Boolean, default: true },
+  },
+  computed: {
+    nuIconButtonClass: function() {
```
Comment (ahmetceylan) [inline src/components/VxpIconButton.vue]: replace "nu" prefix with "vxp"

File: docs/components/vxpiconbutton.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+
+```html
+  <FlexboxLayout class="nubutton-doc-container" flexDirection="column">
+    <FlexboxLayout
+      width="250"
+      class="nubutton-doc-container__flexboxLayout"
+      flexDirection="row"
+      justifyContent="space-around"
+    >
+      <IconButton
```
Comment (ahmetceylan) [inline docs/components/vxpiconbutton.md]: use VxpIconButton instead of IconButton

**Pull request merged**